docs/doxygen/images/http_demo_s3_download.pu

```diff
@@ -0,0 +1,72 @@
+@startuml
```

Comment:
Can you add `skinparam dpi 300` to all of the images and regenerate so that they are not blurry?